### ISSUE TITLE
Allow for keyboard navigation of search results

### ISSUE THREAD

**Issue by bloudermilk** [climatescape/climatescape.org#67]
Visitors should be able to use keyboard navigation to access search results:

* down arrow - highlight the next result, first if not yet pressed, or last if already at end
* up arrow - highlight the previous result, or none if at top
* enter - navigate to the selected result
